website/siteConfig.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// See https://docusaurus.io/docs/site-config for all the possible
+// site configuration options.
+
+const siteConfig = {
+  title: 'Redux Starter kit', // Title for your website.
+  tagline: 'Redux Starter kit Documentation',
+  url: 'https://redux-starter-kit.js.org', // Your website URL
+  baseUrl: '/',
+  docsUrl: '',
+  algolia: {
+    apiKey: '2d058d216b7fd5d68d481fd48ee72c06',
+    indexName: 'redux-starter-kit',
+    algoliaOptions: {}
+  },
+
+  // Used for publishing and more
+  projectName: 'redux-starter-kit',
+  organizationName: 'reduxjs',
+  // For top-level user or org sites, the organization is still the same.
+  // e.g., for the https://JoelMarcey.github.io site, it would be set like...
+  //   organizationName: 'JoelMarcey'
+
+  // For no header links in the top nav bar -> headerLinks: [],
+  headerLinks: [
+    { doc: 'introduction/quick-start', label: 'Quick Start' },
+    {
+      doc: 'using-redux-starter-kit/connect-mapstate',
```

Comment:
Should point to whatever our actual first Markdown page is

Reply:
so basically we just have 1 now,  `quick-start.md`. so remove all header links and leave only quick-start?